scripts/benchmarks/benches/result.py

```diff
@@ -18,7 +18,7 @@ class Result:
     stdout: str
     passed: bool = True
     unit: str = ""
-    explicit_group: str = ""
+    explicit_group: str = "Ungrouped"
```

Comment:
The html output interprets anything other than "" as a group (see https://github.com/oneapi-src/unified-runtime/blob/main/scripts/benchmarks/output_html.py#L117). And every explicit group is shown together on a bar chart.
So this needs to stay as "".
My suggestion is to use "Others" in the markdown output when "" is specified.